diff.html

```diff
@@ -1,33 +1,434 @@
+    .jsondiffpatch-delta > ul > li > ul > li > ul > li.jsondiffpatch-unchanged,
+    .jsondiffpatch-delta > ul > li > ul > li > ul > li > ul > li.jsondiffpatch-unchanged {
+        display: none;
+        visibility: hidden;
+    }
+
+    /* Display identifiers to get some context. */
+    .jsondiffpatch-unchanged[data-key="enemyName"],
+    .jsondiffpatch-unchanged[data-key="modName"],
+    .jsondiffpatch-unchanged[data-key="blueprintName"],
+    .jsondiffpatch-unchanged[data-key="itemName"],
+    .jsondiffpatch-unchanged[data-key="objectiveName"],
+    .jsondiffpatch-unchanged[data-key="bountyLevel"],
+    .jsondiffpatch-unchanged[data-key="keyName"],
+    .jsondiffpatch-unchanged[data-key="tier"],
+    .jsondiffpatch-unchanged[data-key="relicName"],
+    .jsondiffpatch-unchanged[data-key="state"] {
+        display: block;
+        visibility: visible;
+    }
+
+    /* Fix text size in subnodes. */
+    .jsondiffpatch-delta * {
+        font-family: 'Bitstream Vera Sans Mono', 'DejaVu Sans Mono', Monaco, Courier, monospace !important;
+        font-size: 12px !important;
+    }
+
+    /* Display unchanged nodes using the normal text color. */
+    .jsondiffpatch-unchanged,
+    .jsondiffpatch-movedestination {
+      color: inherit;
+    }
+
+    /* Fix excessive margin in subnodes. */
+    .jsondiffpatch-node-type-object > li {
+        margin-bottom: 0 !important;
+    }
+
+    .footer {
+        position: fixed;
+
+        bottom: 0;
+        left: 0;
+        right: 0;
+
+        background: white;
+        color: #555;
+
+        font-size: 12px;
+
+        text-align: center;
+    }
+
+    .footer a {
+        color: #555;
+        text-decoration: none;
+        font-weight: bolder;
+
+        transition: color 0.5s ease;
+    }
+
+    .footer a:hover {
+        color: #0099FF;
+    }
+</style>
+
+<div id="no-script" style="display: flex;">
+    <p>
+        <i class="fa fa-warning"></i><br />
+        This website requires JavaScript.
+    </p>
+</div>
+
+<div id="loading" style="display: none;">
+    <p>
+        <i class="fa fa-refresh fa-spin"></i><br />
+        Loading, please wait...
+    </p>
+</div>
+
+<section>
+    <div class="container">
+        <div class="header">
+            <div>
+                <img src="misc/logo.png" height="64">
+                <h5>Warframe Drop Data - Compare Versions</h5>
+                <a href="http://drops.warframestat.us/">Drop Data</a> | <a target="_blank" href="https://github.com/WFCD/warframe-drop-data#api-endpoints">API Documentation</a> | <a target="_blank" href="https://github.com/WFCD/warframe-drop-data">Github</a>
+            </div>
+            <div id="builds-selection">
+                <p>Compare <select id="builds-old" disabled="disabled"><option value="">Please select a build...</option></select> with <select id="builds-new" disabled="disabled"><option value="">Please select a build...</option></select></p>
```

Comment:
The default/empty option should also have an "disabled" attribute

Reply:
Added.